### PR TITLE
Add a WMSDatePicker widget

### PR DESCRIPTION
**Pull request by tsauerwein**
Works with time mode ``single`` (a single datepicker is shown) and ``range`` (two synchronized datepickers are shown to select a range).

gmf-datepicker

### CODE REVIEW

File: core/examples/themes.js [hunk 1]
```diff
@@ -5,98 +5,116 @@ App.themes = {
...
-                        "name": "road",
-                        "isChecked": true,
-                        "legend": true,
-                        "legendRule": "Roads"
-                    }]
-                }]
-            }, {
-                "name": "fedlimit",
+                "name": "osm_time",
+                "icon": "http://geomapfish.demo-camptocamp.com/pgiraud/wsgi/mapserv_proxy?SERVICE=WMS&VERSION=1.1.1&REQUEST=GetLegendGraphic&LAYER=osm_time&FORMAT=image/png&TRANSPARENT=TRUE&RULE=Dans les temps",
```
Comment (asaunier) [inline core/examples/themes.js]: Shouldn't you rather use http://geomapfish.demo-camptocamp.com/1.6/mapserv_proxy ?

File: core/src/script/CGXP/locale/fr.js [hunk 2]
```diff
@@ -181,6 +181,10 @@ GeoExt.Lang.add("fr", {
         datemonthlabelText: "m/Y",
         datelabelText: "d/m/Y",
         datetimelabelText: "d/m/Y H:i:s",
+        datepickerrangeText: "Veuillez sélectionner une date de début et de fin <br>" +
+            "dans la période de {from} à {to}.",
```
Comment (asaunier) [inline core/src/script/CGXP/locale/fr.js]: du {from} au {to}

File: core/src/script/CGXP/locale/fr.js [hunk 3]
```diff
@@ -181,6 +181,10 @@ GeoExt.Lang.add("fr", {
...
+        datepickersingleText: "Veuillez sélectionner une date dans la " +
+            "période de {from} à {to}.",
```
Comment (asaunier) [inline core/src/script/CGXP/locale/fr.js]: du {from} au {to}

**Comment by asaunier** [comment]
Except a few minor issues, looks good to me :)

**Comment by tsauerwein** [comment]
Thanks for your comments!

**Pull request merged**